LiteNetLibSystemExample/core/BaseEntityPawn.cs

```diff
@@ -3,28 +3,28 @@
 using LiteEntitySystem.Extensions;
 using System.Collections.Generic;
 
-[UpdateableEntity(true)]
+//[UpdateableEntity(true)]
```

Comment:
This attribute makes sure that entities, even if not owned by the client, still get updated in the protected override void Update method. Since it’s commented out, though, it breaks the interpolation behavior. This means that when more than three instances are running (1 server, 2 clients), we can't see the movement from the other client.

The equivalent attribute in the latest LiteEntitySystem is `EntityFlags`

```suggestion
[EntityFlags(EntityFlags.UpdateOnClient)]
```